### ISSUE TITLE
[A11y][ARIA]NVDA/Narrator is reading about the data which is not present on the screen at the table "Version History"

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8196]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188406
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450) 

## Repro Steps:

    Turn on NVDA
    Hit the URL and login with appropriate credentials
    Tab till "Profile" button in the menu bar and press enter
    Tab till any Manage packages"  and press enter.
    Tab till search bar and search for 'Json' and press enter 
    Tab till the first result "Newtonsoft.Json" and press enter 
    Tab till "Version" of any result 
    Tab till "Version history" and press enter to expand
    Verify whether the NVDA is announcing the blank columns and allows navigation with CTRL + ALT + Arrow keys

## Actual Results:

NVDA/Narrator is reading about the data which is not present on the screen at the table "Version History"

   
## Expected Results:
Data should be visible on the screen.

Narrator/NVDA should read correct data which is present in the table and which is visible on the screen

Note: when data is not present in any column it would be better if screen reader announce it as empty cell.

## User Impact:

Screen reader users may not be understand about the columns in the table and also get confused with navigation 

## Notes
In a surprise twist, we are reading out too much information. This seems most likely connected to the fact that the table has columns that are not visible at all times (status, signature information, deprecation information). For cases where this information isn't present, it should not be in the table either (if it should never be presented). If it could possibly be presented, but it isn't filled in, we should indicate that the cell is empty.
